loops-and-arrays.js

```diff
@@ -57,5 +71,8 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 11 is 1331'
```

Comment:
Het berekenen van het volume door een getal tot de macht drie te verheffen wordt correct gedaan. De string interpolatie is duidelijk en functioneel.